gef-ui/server/types/express-session.d.ts

```diff
@@ -0,0 +1,18 @@
+import { ReportPeriod } from '@ukef/dtfs2-common';
+import { PortalSessionUser } from './portal-session-user';
+
+export type UserSessionData = {
```

Comment:
do we want to commonise these user session types, and then export one for portal and one for gef?

Reply:
I looked into doing this and had a chat with Josh. We decided not to commonise because there are several `SessionUser` objects and they are service dependent (dtfs-central-api has a different session user to portal).
Whilst portal & gef-ui will have the same sessionUser because theyre the same thing, I think I'm happy with this being duped?

Reply:
I think we should look to dedupe this or well run into inconsistances pretty fast  happy to talk about this tho